src/utils.jl

```diff
@@ -2,10 +2,17 @@ export idx, ising, proj
 export HadamardMPS, rq
 export all_states, local_basis, enum, state_to_ind, rank_vec
 export @state
+export reshape_row
 
 using Base.Cartesian
 import Base.Prehashed
 
+function reshape_row(A::AbstractArray{T}, dims::Tuple) where {T <: Number}
```

Comment:
Why is this here? Is it used in the code somewhere?

Reply:
removed